addons/project/views/project_sharing_views.xml

```diff
@@ -132,6 +132,13 @@
                             attrs="{'invisible' : &quot;[('user_ids', 'not in', [uid])]&quot;}" data-hotkey="q"/>
                     <field name="stage_id" widget="statusbar" options="{'clickable': '1', 'fold_field': 'fold'}" attrs="{'invisible': [('project_id', '=', False), ('stage_id', '=', False)]}" />
                 </header>
+                <div groups="base.group_user" class="alert alert-info alert-dismissible rounded-0 fade show d-print-none css_editable_mode_hidden">
```

Comment:
```suggestion
                <div groups="base.group_user" role="status" class="alert alert-info alert-dismissible rounded-0 fade show d-print-none css_editable_mode_hidden">
```
Didn't see the runbot, looks like there is a `role` attribute missing here. But are the alert classes really needed/appropriate here?

Reply:
i know it's not cohrent to have alert-* classes but it's the only way to get the same UI as they use it to display the info we're extending to the new view